cache.js

```diff
@@ -61,7 +61,7 @@ define(function () {
 		// end browser.js adapters
 	}
 
-	if (typeof window !== 'undefined' && window.sessionStorage !== 'undefined') {
+	if (typeof window !== 'undefined' && typeof window.sessionStorage !== 'undefined') {
```

Comment:
Would `window.sessionStorage != null` make more sense, to check that it's neither `null` nor `undefined`?